src/relation_engine_server/utils/arango_client.py

```diff
@@ -38,6 +38,8 @@ def run_query(query_text=None, cursor_id=None, bind_vars=None, batch_size=100):
         method = 'POST'
         req_json['count'] = True
         req_json['query'] = query_text
+        if full_count:
+            req_json['options'] = {'fullCount': True}
```

Comment:
Interesting.. I hadn't seen that feature in Arango. Should be really useful if we can keep the queries efficient.

Reply:
BTW the the really low cyclomatic complicity limit drives me nuts on this repo 🤦‍♂️

Reply:
Feel free to remove it or increase it if you find it to be a pain.

To me it is a great prompt to try to see if I can write code without new execution paths, or at the very least split up the function. For example, in this case you could do `req_json['options'] = {'fullCount': full_count}`. Maybe not the most important example.